packages/executable/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "executable",
+  "version": "1.0.0",
+  "description": "Build Serge as executable",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "find": "^0.3.0",
+    "pkg": "^4.4.0"
+  },
+  "bin": "../server/bin.js",
+  "pkg": {
+    "assets": [
+      "../client/build/**/*",
```

Comment:
@IanMayo We can leave this in for now but I'd rather us not reference other packages in the monorepo as relative paths. I had to do the same thing in the `server` package so like I say, it's fine for now but we should add refactoring these references out to our tech debt.